doc/manuals/devel/sourceCode.md

```diff
@@ -382,8 +383,61 @@ This library is similar to the [**ngsi9**](#srclibngsi9) and [**ngsi10**](#srcli
 
 ## src/lib/mongoBackend/
 
-The most important of all libraries of the broker, the **mongoBackend** library is where all the database interaction takes place. This library is described in detail in [a separate document](mongoBackend.md).
+The most important of all libraries of the broker, the **mongoBackend** library is where all the database interaction takes place (through the "wrapping" library **mongoDriver**). This library is described in detail in [a separate document](mongoBacDriver).
```

Comment:
Broken link?

Reply:
Fixed in 102de34d8ce2ff652aaf714bbd5a2da535432a9b